Stresser/EtwEventsGuid.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+#include "EtwEventTypes.h"
+#include "EventProvidersGuid.h"
+#include <map>
+
+
+const std::map<EtwEventTypes, GUID> EtwEventsGuid =
+{
+	{EtwEventTypes::DiskIo, DiskIoGuid},
+	{EtwEventTypes::DiskFileIo, FileIoGuid},
+	{EtwEventTypes::FileIo, FileIoGuid},
+	{EtwEventTypes::FileIoInit, FileIoGuid},
+	{EtwEventTypes::Registry, RegistryGuid},
+	{EtwEventTypes::Process, ProcessGuid},
+	{EtwEventTypes::ProcessCounters, ProcessGuid}
+};
```

Comment:
I think this all file can go under the header: `EtwEventTypes.h`.
They are very related.